src/app/(pages)/map/page.tsx

```
@@ -0,0 +1,396 @@
+          position: 'fixed',
+          bottom: '80px',
+          left: 0,
+          right: 0,
+          height: 'calc(100vh - 312px)',
+        }}
+      >
+        <iframe
+          src={`https://salvador-airport-preview.blumaps.com.br/`}
+          style={{ width: '100%', height: '100%' }}
+        />
+      </Box>
```

Biome (1.9.4)

[error] Provide a title attribute when using iframe elements.

Screen readers rely on the title set on an iframe to describe the content being displayed.

(lint/a11y/useIframeTitle)